Assets/Scripts/Characters/TurnController.cs

```diff
@@ -1,22 +1,18 @@
-using System;
 using System.Collections.Generic;
+using Mirror;
 using UI;
 using UnityEngine;
-using UnityEngine.InputSystem;
 using Utils;
 
 namespace Characters
 {
-    public class TurnController : MonoBehaviour
+    public class TurnController : NetworkBehaviour
```

Comment:
The turn controller is spawned by the NetworkManager now. I think that'll be typical for most level wide state objects like this